dbt_copilot_helper/commands/copilot.py

```diff
@@ -137,11 +137,12 @@ def _normalise_keys(source: dict):
 
 
 @copilot.command()
-def make_addons():
+@click.option("-d", "--manifest-dir", type=str, default=".")
```

Comment:
Shall we change this one as well? I believe this command is to generate the addons files for each environment from the root directory.